ios/RNJudo.m

```diff
@@ -65,34 +93,101 @@ - (dispatch_queue_t)methodQueue
     resolve([NSNumber numberWithBool:canUse]);
 }
 
-RCT_REMAP_METHOD(makeApplePayPayment,
+RCT_REMAP_METHOD(makeAppleNativePayment,
```

Comment:
What's this obsession with "Native"? It's called Apple Pay

Reply:
👍